source/_integrations/tuya.markdown

```diff
@@ -110,22 +129,22 @@ Click the created project to enter the `Project Overview` page and get the `Auth
 {% configuration_basic %}
 
 "1004: sign invalid":
-  description: Incorrect Access ID or Access Secret. Please refer to [Tuya credential table](https://github.com/tuya/tuya-home-assistant/wiki/Install-Tuya-v2?_source=d10de34623e3daca5b02e3c31528a0c4#3-enter-your-tuya-credential).
+  description: Incorrect Access ID or Access Secret. Please refer to the **Configuration** part above.
 
 "1106: permission deny":
   description: >
     - App account not linked with cloud project: On the [Tuya IoT Platform](https://iot.tuya.com/cloud/), you have linked devices by using Tuya Smart or Smart Life app in your cloud project. For more information, see [Link devices by app account](https://developer.tuya.com/en/docs/iot/Platform_Configuration_smarthome?id=Kamcgamwoevrx&_source=7a356dd493196a01bb9021b7680a2a45#title-3-Link%20devices%20by%20app%20account).
 
     - Incorrect username or password: Enter the correct account and password of the Tuya Smart or Smart Life app in the **Account** and **Password** fields. Note that the app account depends on which app (Tuya Smart or Smart Life) you used to link devices on the [Tuya IoT Platform](https://iot.tuya.com/cloud/).
 
-    - Incorrect availability zone: See [Availability Zone](https://github.com/tuya/tuya-home-assistant/wiki/Tuya-IoT-Platform-Configuration-Guide-Using-Smart-Home-PaaS#region--available-zone-correspondence) and select the correct availability zone.
+    - Incorrect country. You must select the region of your account of the Tuya Smart app or Smart Life app.
 
 "1100: param is empty":
-  description: Empty parameter of username or app. Fill the parameters refer to [Tuya credential table](https://github.com/tuya/tuya-home-assistant/wiki/Install-Tuya-v2?_source=d10de34623e3daca5b02e3c31528a0c4#3-enter-your-tuya-credential).
+  description: Empty parameter of username or app. Please fill the parameters refer to the **Configuration** part above.
```

Comment:
This is from the `current` branch and should not be applied to the `next branch again.

Reply:
ok, I thought the next branch will replace current branch content, so I update the next branch content too. So I should create a PR to current branch for the new changes?

Reply:
branches are merged, not replaced.

Reply:
OK, just revert the changes in `current` branch.